src/statusBar.ts

```diff
@@ -55,6 +55,19 @@ class StatusBarImpl implements vscode.Disposable {
     this._statusBarItem.dispose();
   }
 
+  public Get() {
+    let text = this._statusBarItem.text;
+    return text;
+  }
+
+  public GetTrimmed() {
```

Comment:
Where are you consuming this? This is also highly coupled with the status bar completion code -- I don't think this logic belongs here.

Reply:
moved this to actions.ts CommandNavigateInCommandlineOrSearchMode class